### PR TITLE
replace quotation marks

### PR DESCRIPTION
**Pull request by meinradandermatt**
# Motivation
On paste the quotation marks in a text should be replaced with quotation marks set in the config. 

# Changelog
- 🎁 Add quotation mark replace on paste
added config in `pastedHtmlRules`
```
replaceQuotes: {
      quotes: ['“', '”'],
      singleQuotes: ['‘', '’'],
      apostrophe: '’'
    }
```

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -75,6 +75,12 @@ export default {
     // and content (text content and child elements) will get removed.
     blacklistedElements: ['style', 'script'],
 
-    keepInternalRelativeLinks: false
+    keepInternalRelativeLinks: false,
+
+    replaceQuotes: {
```
Comment (peyerluk) [inline src/config.js:80]: I think we should not introduce a default config.

**Review by peyerluk — APPROVED** [review]

File: src/clipboard.js [hunk 2]
```diff
@@ -168,3 +170,11 @@ export function cleanWhitespace (str) {
       : ' '
     ))
 }
+
+export function replaceAllQuotes (str) {
+  if (replaceQuotes.quotes) {
```
Comment (peyerluk) [inline src/clipboard.js:175]: Can you alsoadd `if (replaceQuotes &&)` here so we can deal with a completely undefined config?

Comment (meinradandermatt) [reply src/clipboard.js]: it's always an empty object https://github.com/livingdocsIO/editable.js/pull/212/files#diff-e4b7db0db2bb9f545c59492bc99ef442cd32bc5402335021236b8f159dd3cd06R22

Comment (peyerluk) [reply src/clipboard.js]: perfect!

**Pull request merged**